operations/rack_resiliency/rack_resiliency.md

```diff
@@ -0,0 +1,490 @@
+
+###### 4.1.1 Control Loop
+
+This loop ensures the monitoring infrastructure is properly initialized and maintained. It performs the following tasks:
+
+* Starts the internal application server to receive notifications from HMNFD.
+* Verifies and maintains active subscriptions to HMNFD.
+* Periodically updates the status of critical services and zones when no active monitoring events are in progress.
+
+###### 4.1.2 Monitoring Loop
+
+Triggered upon receiving a notification from HMNFD, this loop performs targeted analysis and response actions, including:
+
+* Determining whether the failure is isolated to a single node or part of a rack-level failure.
+* Monitors the health and status of critical services, and logs alerts if any imbalance is detected following a failure.
+* Verifying placement and availability of management nodes, including both `Kubernetes` control plane and `Ceph` storage nodes.
+
+**Note:** Any modifications to critical services may take up to 10 minutes to reflect in CLI command outputs and to appear in the monitored list.
+
+### 5 RR CLI
+
+The CLI for interfacing with rack resiliency service is part of `craycli`. A new module (`rrs`) is added to `craycli` to support rack resiliency specific commands. The backend for the CLI which serves REST endpoints is provided as a container along with the rack resiliency monitor in the same pod.
+
+#### Commands 
+
+The following commands are supported as two groups:
+
+**zones:** to list / describe `Kubernetes` topology zones and `Ceph` zones as configured
+
+**critical services:** to list/describe critical services defined in the RMS ConfigMap
+
+The sub commands for each of the groups are used to list, describe and update resources associated with the groups.
+
+RRS `craycli` commands
+
+```bash
+    cray rrs zones list
+    cray rrs zones describe 
+    cray rrs criticalservices list
+    cray rrs criticalservices describe
+    cray rrs criticalservices update --from-file <file-path>
+```
+
+Please refer https://github.com/Cray-HPE/docs-csm/blob/CASM-5612/operations/rack_resiliency/rack_resiliency_cli.md for more information on viewing/managing zones (`Kubernetes` and `Ceph`) and critical service via RR CLI.
+
+## Steps to achieve RR
+
+### Install/ Upgrade
+
+#### 1 To enable Rack Resiliency feature
+
+By default the Rack Resiliency feature will be disabled and the user/admin can enable the Rack Resiliency feature by following the below steps.
+
+**Warning:** The Rack Resiliency feature flag has to be enabled before the CSM installation or upgrade is performed.
+Enabling this feature will create zones for the master, worker and storage management nodes.
+
+**Procedure**
+
+Update the `customizations.yaml` file before commencing the CSM installation or upgrade.
+
+Update the `spec.services.rack-resiliency.enabled` flag from `false` to `true` in the `customizations.yaml` file. This `customizations.yaml` file will be present in the root directory of the CSM tarball.

```

Comment:
Oh, or is this a clarification of the previous step? Since it's a subsequent paragraph, it sounds like it's the next thing to do after editing `customizations.yaml`. This section needs to be reorganized for clarity.